src/platform/utilities/environment/index.js

```diff
@@ -41,7 +29,7 @@ function getEnvironment() {
   let platform;
 
   if (location.port === '3001') {
-    platform = 'local';
+    platform = 'localhost';
   } else if (location.port === `${process.env.WEB_PORT || 3333}`) {
     platform = 'e2e';
   } else {
```

Comment:
This file could use some serious sprucing up. The `location.hostname` and and the `process.env.BASE_URL` pieces are particularly confusing. Apparently, our Nightwatch Docker runs under `http://vets-website`, so we have some dynamic pieces like that.